hushline/templates/premium.html

```diff
@@ -0,0 +1,96 @@
+{% extends "base.html" %}
+{% block title %}Premium{% endblock %}
+{% block content %}
+  <h2>Choose a Hush Line Plan</h2>
+
+  <div id="plan-wrapper">
+    <div class="plan">
+      {% if user.tier_id == 1 or user.tier_id == None %}
```

Comment:
You have `user.tier_id == 1 or user.tier_id == None` a few times. It uses `1` as a magic number, so maybe the `User` should have an `is_free_tier` property?

Reply:
Good idea. Fixed in 0e3d8557c9d60ffbef0a62872889841223831535